cluster/local/integration_tests.sh

```diff
@@ -137,9 +137,9 @@ eval $(make --no-print-directory -C ${projectdir} build.vars)
 
 # ------------------------------
 
-HOSTARCH="${HOSTARCH:-amd64}"
```

Comment:
@muvaf did this get removed in the build submodule?

Reply:
It used to be translated from `x86_64` to `amd64` until https://github.com/upbound/build/pull/139 . It seems now we get `TARGETARCH` for the target architecture we're building for instead of relying on the ARCH of the host. @AaronME can confirm

Reply:
@muvaf but I think we should be running integration tests using the ARCH of host.

Reply:
They end up being the same when not overridden but you're right, using the arch of the host would be more correct here. I'll update it.